### DISCUSSION TITLE
How can I use the source code to create executable jar?

### DISCUSSION THREAD

**Discussion by ChineseUncleTom** [jagrosh/MusicBot#638] [Questions] [answered]
Hi everyone! Recently, I found out the v0.3.3 release is out dated, but the source code has been updated by the contributors.

I tried many way to create the executable jar files like the release one, but they didn't work.

So I am wondering how you guys use the source code to build executable jar?

<3

**Comment by jagrosh** [comment, marked as answer]
As per the notice in the readme [here](https://github.com/jagrosh/MusicBot#editing), building this project yourself is not supported. Please be patient and wait for the next release.

**Reply by ChineseUncleTom** [reply]
Alright. Thanks!
